### PR TITLE
Listings

### PR DESCRIPTION
**Pull request by norbadboy**
All methods in listing routes working

### CODE REVIEW

**Comment by norbadboy** [comment]
Two of the conflicts are resolved. Not sure what to do with package-lock.json.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 # Keep environment variables out of version control
 .env
+prisma
```
Comment (Fermain) [inline .gitignore:4]: I don't think excluding the whole prisma folder is a good idea. That incudes the schema which drives the database.

File: coverage/clover.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Fermain) [inline coverage/clover.xml:1]: coverage should be ignored

**Review by Fermain — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Fermain) [inline .gitignore:4]:
```suggestion
coverage
```

Comment (norbadboy) [reply .gitignore]: I thought we have removed it by resolving the conflict last time, but request on changes has been approved

**Comment by Fermain** [comment]
@norbadboy I have resolved the conflicts and merged a PR to remove the coverage files. You can now see there are only 10 changes down from 40 which is much easier to manage in a PR :)

**Review by Fermain — APPROVED** [review]

**Pull request merged**